powershell-gallery/docs-conceptual/powershellget/supported-repositories.md

```diff
@@ -0,0 +1,283 @@
+---
+description: This article lists the repositories that have been tested with PowerShellGet v3 and how to configure them.
+ms.date: 06/28/2023
+title: Supported repository configurations
+---
+# Supported repository configurations
+
+The **Microsoft.PowerShell.PSResourceGet** module, also known as **PowerShellGet v3**, works with
+NuGet package repositories and local file stores. In general, the cmdlets should work with any NuGet
+repository that supports the NuGet protocol. However, not all NuGet repositories support all the
+features of PSResourceGet.
+
+The following repositories have been tested with PSResourceGet.
+
+## The PowerShell Gallery
+
+The PowerShell Gallery is the default repository for PSResourceGet. The Gallery is a NuGet
+repository that uses the NuGet v2 protocol.
+
+You don't need credentials to search, download, or install packages from the PowerShell Gallery. You
+must create an account and API key to publish packages to the PowerShell Gallery. For more
+information, see [Creating and publishing an item][01].
+
+The PowerShell Gallery supports all the features of PSResourceGet.
+
+## NuGet.org
+
+NuGet.org is a public host of NuGet packages used by millions of .NET and .NET Core developers every
+day. The NuGet.org provides Gallery a NuGet repository that uses the NuGet v3 protocol.
```

Comment:
I'm not sure about the last sentence. Does NuGet.org provide a gallery, which is a NuGet repository, or something else?

```suggestion
NuGet.org is a public host of NuGet packages used by millions of .NET and .NET Core developers every
day. NuGet.org provides a NuGet repository that uses the NuGet v3 protocol.
```